common/test_fiximports.py

```diff
@@ -2,12 +2,16 @@
 from textwrap import dedent
 from twisted.trial import unittest
 
+#returning functions
+def d_setUp():
+    return None
 
 class TestFixImports(unittest.TestCase):
 
     def setUp(self):
         self.fiximports = FixImports()
-        self.fiximports.printErrorMsg = lambda *_: None
+        ret_msg = d_setUp()
```

Comment:
It sounds like that would be more correct; however, I don't think that PEP8 actually requires this change.